src/core/opamConsole.ml

```diff
@@ -824,12 +824,13 @@ let confirm ?(require_unsafe_yes=false) ?(default=true) fmt =
       then
         (formatted_msg "%sn\n" prompt; false)
       else
-        short_user_input ~prompt ~default:(if default then "y" else "n")
+        short_user_input ~prompt ~default:' '
```

Comment:
What is the reason behind removing the pre-written answer?

Reply:
The reason is that it is more standard and expected from regular unix users. All the commands with interactive y/n questions i can think of do this

Reply:
Fair enough, i don't have a strong opinion on that :) lgtm